packages/manager/modules/advices/src/advices/template.html

```diff
@@ -0,0 +1,17 @@
+<oui-tile
+    data-ng-if="!$ctrl.loading"
+    heading="{{ ::'advices_heading' | translate }}"
```

Comment:
The `heading` is still the same for each groups ?

Reply:
As of now, there is only one advice group in all services. We have added an advice group to make it future proof. Yes, the tile heading is always same.